backup.sh

```diff
@@ -15,23 +16,24 @@ copy_s3 () {
   echo "Uploading ${DEST_FILE} on S3..."
   if [[ -z "$S3_PREFIX" ]]; then
     # backup without prefix
-    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$DEST_FILE && \
-    aws $AWS_ARGS s3 cp s3://$S3_BUCKET/$DEST_FILE s3://$S3_BUCKET/$DEST_LATEST_FILE
+    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$FREQUENCY/$DEST_FILE && \
+    aws $AWS_ARGS s3 cp s3://$S3_BUCKET/$FREQUENCY/$DEST_FILE s3://$S3_BUCKET/$FREQUENCY/$DEST_LATEST_FILE
   else
-    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$S3_PREFIX/$DEST_FILE && \
-    aws $AWS_ARGS s3 cp s3://$S3_BUCKET/$S3_PREFIX/$DEST_FILE s3://$S3_BUCKET/$S3_PREFIX/$DEST_LATEST_FILE
+    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$S3_PREFIX/$FREQUENCY/$DEST_FILE && \
+    aws $AWS_ARGS s3 cp s3://$S3_BUCKET/$S3_PREFIX/$FREQUENCY/$DEST_FILE s3://$S3_BUCKET/$S3_PREFIX/$FREQUENCY/$DEST_LATEST_FILE
   fi
   if [ "$?" == "0" ]; then
     rm -f $SRC_FILE
   else
-    echo "Error uploading ${DEST_FILE} on S3"
+    echo "Error uploading ${FREQUENCY} backup file ${DEST_FILE} on S3"
   fi
 }
-
+FREQUENCY=$1
```

Comment:
@vijayabharathib I think this is not needed. Because we already define that in line 7.

Reply:
line number 7 is for "copy s3" method alone. we have this outside in global scope. in fact we are pssing this as $3 to copy_s3 method.

Reply:
Got it.